code/controllers/subsystem/ticker.dm

```diff
@@ -57,6 +57,11 @@ SUBSYSTEM_DEF(ticker)
 	var/mode_result = "undefined"
 	var/end_state = "undefined"
 
+	//Sunset - Crew Objectives
```

Comment:
can be modularized in `sunset/code/controllers/subsystem/ticker.dm`

like
```
/datum/controller/subsystem/ticker
	var/list/successfulCrew = list()
	var/list/crewobjlist = list()
	var/list/crewobjjobs = list()
```

Reply:
I have created ticker under sunset and have modularized this portion. Thank you for catching that.